addons/website/static/src/xml/website.backend.xml

```diff
@@ -30,24 +30,24 @@
 
     <t t-name="website.plausible_content">
         <div class="row o_dashboard_visits" t-if="widget.groups.website_designer">
-            <div class="col-12 o_box">
-                <h2>Analytics</h2>
-                <div t-if="widget.dashboards_data.plausible_share_url" class="row">
+            <div t-if="widget.dashboards_data.plausible_share_url" class="col-12">
+                <div class="container">
                     <div class="ratio ratio-1x1">
-                        <iframe t-att-src='widget.dashboards_data.plausible_share_url' class="embed-responsive-item" frameborder="0" loading="lazy"/>
+                        <iframe t-att-src='widget.dashboards_data.plausible_share_url' class="embed-responsive-item" frameborder="0" loading="lazy" scrolling="no"/>
```

Comment:
This `scrolling` is not useful since you hardcoded some height, no?
I doubt I left it, if I missed a good reason to have it. If it can be removed, maybe do when investigating if the hardcoded height can be replaced by something else :+1: